packages/style-value-parser/src/at-queries-from-tokens/__tests__/media-query-test.js

```diff
@@ -1028,22 +1035,32 @@ describe('Test CSS Type: @media queries', () => {
     expect(MediaQuery.parser.parseToEnd(query).toString()).toEqual(query);
   });
 
-  test.skip('@media (width: calc(100% - 20px))', () => {
-    expect(
-      MediaQuery.parser.parseToEnd('@media (width: calc(100% - 20px))'),
-    ).toMatchInlineSnapshot();
-  });
-
-  test.skip('@media (min-width: calc(300px + 5em))', () => {
-    expect(
-      MediaQuery.parser.parseToEnd('@media (min-width: calc(300px + 5em))'),
-    ).toMatchInlineSnapshot();
+  test('@media (min-width: calc(300px + 5em))', () => {
```

Comment:
Do we have any tests for when this MQ is malformed with no spaces around the operator within `calc`?